basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,130 @@
 const helpers = require('./project-1');
+test('should run test', () => {
+
+})
 
 // start testing!
+it('should equal 20', () => {
+    const actual = helpers.multiplyByTen(2)
+
+    expect(actual).toEqual(20)
+})
+
+it('should equal -3', () => {
+    const expected = 2
+    const actual = helpers.subtractFive(expected)
+
+    expect(actual).toEqual(-3)
+})
+
+it('should equal same length', () => {
```

Comment:
Change the wording to say what the test should return and why. Something like, `it('should return true when two strings have the same length')`